op-node/rollup/engine/events.go

```diff
@@ -323,6 +323,27 @@ func (d *EngDeriver) OnEvent(ev event.Event) bool {
 			}
 		} else {
 			d.log.Info("successfully processed payload", "ref", ref, "txs", len(x.Envelope.ExecutionPayload.Transactions))
+			payload := x.Envelope.ExecutionPayload
+			if payload != nil {
+				latestBlockTimeStamp := uint64(payload.Timestamp)
+
+				currentTime := uint64(time.Now().Unix())
+
+				if latestBlockTimeStamp <= currentTime {
+					timeDiff := currentTime - latestBlockTimeStamp
+
+					if timeDiff < 2 {
```

Comment:
Could we use the blocktime variable rather than hardcode 2? Seems more flexible this way